src/internal/operators/pluck.ts

```diff
@@ -39,20 +39,20 @@ export function pluck<T, K1 extends keyof T, K2 extends keyof T[K1], K3 extends
  * @see {@link map}
  *
  * @param {...string} properties The nested properties to pluck from each source
- * value (an object).
+ * value.
  * @return {Observable} A new Observable of property values from the source values.
  * @method pluck
  * @owner Observable
  */
-export function pluck<T, R>(...properties: string[]): OperatorFunction<T, R> {
+export function pluck<T, R>(...properties: Array<string | number>): OperatorFunction<T, R> {
   const length = properties.length;
   if (length === 0) {
     throw new Error('list of properties cannot be empty.');
   }
   return (source: Observable<T>) => map(plucker(properties, length))(source as any);
 }
 
-function plucker(props: string[], length: number): (x: string) => any {
+function plucker(props: Array<string | number>, length: number): (x: string) => any {
```

Comment:
I realize that this isn't code you wrote, but `string` is not the correct type for `x` here. Could you change it to `any`?